9.2-2.3/alpine/Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM postgres:9.2-alpine
+MAINTAINER Régis Belson <me@regisbelson.fr>
+
+ENV POSTGIS_VERSION 2.3.1
+ENV POSTGIS_SHA256 0f1e0fe08f55b6db791d8600bb343e4f77590725acbbec99094a30b8b7439cd8
+
+RUN echo http://dl-cdn.alpinelinux.org/alpine/edge/main >> /etc/apk/repositories
```

Comment:
Well, I tried to reproduce the problem but it's working now without the `apk upgrade`.
Just to answer your question, here's the difference in size with and without:
```
REPOSITORY                     TAG                     IMAGE ID            CREATED             SIZE
mdillon/postgis                9.2-alpine-upgrade      62412cc45521        4 minutes ago       140 MB
mdillon/postgis                9.2-alpine-no-upgrade   5580d6c1f7a1        7 minutes ago       133 MB
```